data/coalition/coalition jobs.txt

```diff
@@ -1328,3 +1328,23 @@ mission "Coalition: From Games 3"
 		payment
 		payment 25000
 		dialog phrase "coalition games dialog"
+
+
+mission "Heliarchs Buy Jump Drive Job"
+	job
+	repeat
+	name "Sell Jump Drive"
+	description "Sell a jump drive to the Heliarchs for five million credits."
+	source
+		government "Heliarch"
+	to offer
+		not "Coalition: Lost: Offer: active"
+		not "Heliarchs Buy Jump Drive Mission: declined"
+		has "Heliarchs Buy Jump Drive Mission: offered"
```

Comment:
I don't think declining the `landing` mission should necessarily preclude a person from being able to take up this opportunity later through the job board.
How about instead, the defer and decline options in the mission skip selling the drive there but the Heliarchs say they'll "a notice in the job board leave, if your mind you ever change."

You could perhaps use an intermediate job that has the more detailed text from the mission, subsequently offering only this one with the dialog.

Reply:
I think it'll be just the one refuse message in that case, don't see much a point to keeping a "no" and also a "no, never" if the point is they add the job anyways if you change your mind.

For the intermediate job, you mean something like having jobs 1 and 2, job 1 offers if you decline, and includes the main convo, while 2 is right after you first accept or after doing 1?

Reply:
> For the intermediate job, you mean something like having jobs 1 and 2, job 1 offers if you decline, and includes the main convo, while 2 is right after you first accept or after doing 1?

Yes

Reply:
@warp-core added an intermediate job and the interaction from the mission directing you to the Job Board in case you decline.